.travis.yml

```diff
@@ -21,12 +21,22 @@ before_install:
   - sudo rm -rf /usr/local/.git/refs /usr/local/.git/packed-refs
   - sudo rsync -az "$TRAVIS_BUILD_DIR/" /usr/local/
   - export TRAVIS_BUILD_DIR="/usr/local"
-  - sudo chown -R $USER /usr/local
-  - env | grep TRAVIS_
-  - if [ -f ".git/shallow" ]; then travis_retry git fetch --unshallow; fi
+  - if [ -f ".git/shallow" ]; then
+      travis_retry git fetch --unshallow;
+    fi
   - git reset --hard $TRAVIS_COMMIT
   - git clean -qxdff
 
+install:
+  - if [ "$HOMEBREW_OLD_RUBY_TEST" = "1.8.7" ]; then
+      brew install --build-bottle homebrew/versions/ruby187;
```

Comment:
Pretty sure the bottle is broken as build from source it's installing outside the sandbox